File: omnes.Web/Modules/ServerTypes/Parametros/PaisesForm.ts
import { StringEditor, BooleanEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface PaisesForm {
    NombrePais: StringEditor;
    Baja: BooleanEditor;
}

export class PaisesForm extends PrefixedContext {
    static formKey = 'Parametros.Paises';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!PaisesForm.init)  {
            PaisesForm.init = true;

            var w0 = StringEditor;
            var w1 = BooleanEditor;

            initFormType(PaisesForm, [
                'NombrePais', w0,
                'Baja', w1
            ]);
        }
    }
}

File: omnes.Web/Modules/Parametros/Paises/PaisesForm.cs
using Serenity.ComponentModel;

namespace omnes.Parametros.Forms;

[FormScript("Parametros.Paises")]
[BasedOnRow(typeof(PaisesRow), CheckNames = true)]
public class PaisesForm
{
    public string NombrePais { get; set; }
    public bool Baja { get; set; }
}